tests/input/resources.yaml

```diff
@@ -138,6 +63,28 @@ simulation_resources:
         source_uri: "file://path/to/file"
       - filename: the_name_of_second_file.json
         source_uri: "file://path/to/second/file"
+    input:
+      ss3:AluminiumMaterialCard:
+        - function:
+            functionType: application/vnd.dlite-generate
+            configuration:
+              driver: plugin_abaqus_material
+              # the name of the file is as expected by the Abaqus input template
+              location: Section_materials.inp
+              datamodel: http://www.sintef.no/calm/0.1/AluminiumMaterialCard
+    output:
+      ss3:AbaqusDeformationHistory:
+        - function:
+            functionType: application/vnd.dlite-convert
+            configuration:
+              module_name: ss3_wrappers.abaqus_convert
+              function_name: abaqus_converter
+              inputs:
+                - label: cement_output
+                  datamodel: http://onto-ns.com/meta/2.0/core.singlefile
+              outputs:
+                - label: cement_output_instance
```

Comment:
I think that `reinforced_concrete_instance` would be a better label.

Reply:
Dummy example, not really imortant to ontoconv. If you wish to make the tests more realistic wrt SS3 it is possible, but not something I want ot prioritise now, as SS3 is still not finalised

Reply:
Ok, remember it for SS3